### PR TITLE
MM-13701: E2E test consecutive message does not repeat profile info

### PR DESCRIPTION
**Pull request by manland**
#### Summary

Consecutive message does not repeat profile info

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/10085

#### Checklist

- [X] Ran `make check-style` to check for style errors (required for all pull requests)
- [X] Ran `make test` to ensure unit and component tests passed
- [X] Added or updated unit tests (required for all new features)

### CODE REVIEW

**Comment by hanzei** [comment]
Hey @manland,

Thank you very much for the PR! :+1: 

I looks like there are some code style problems. You can see them at https://build.mattermost.com/blue/organizations/jenkins/mw%2Fmattermost-webapp/detail/PR-2326/1/pipeline. `make check-style` runs all the style checks, so you can test on your local dev environment.

**Comment by saturninoabril** [comment]
Thanks @manland for your contribution!
I've tried to run it but it's failing on me, maybe because there's some messages already posted in a channel.
I'm thinking to simplify the flow in checking this test case, something like:
1. Login as another user, could be "sysadmin" and post a message
2. Login as "user-1" and post three consecutive messages
3. Check post list content
 - Check that the first message is posted. Check that the image placeholder and the profile image are visible.
 - Check that the second message is posted. Check that the image placeholder is visible but without profile image.
 - Check that the third message is posted. Check that the image placeholder is visible but without profile image.

Let me know if you need help.

**Comment by manland** [comment]
Thx @hanzei I have fixed issues, It's really hard to switch my mind from callback `then()` to inline... 

Thx @saturninoabril I have rework test to be more "cypress style" without callback hell. I don't know if its exactly what you had in mind... Tell me if I need to change something!

File: cypress/integration/channel/message_spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+describe('Message', () => {
+    it('should show only 1 header when user send multiple posts', () => {
+        // 1. Login as sysadmin and go to /
+        cy.login('sysadmin');
+        cy.visit('/');
+
+        // 2. Post a message to force next user message to display a message
+        cy.get('#post_textbox').type('Hello').type('{enter}');
+
+        // 3. Login as sysadmin and go to /
```
Comment (saturninoabril) [inline cypress/integration/channel/message_spec.js]:
```suggestion
        // 3. Login as "user-1" and go to /
```

**Comment by jespino** [comment]
Thank you @manland!

**Pull request merged**